### PR TITLE
Bracketing unit tests

### PR DESCRIPTION
**Pull request by Vilin97**
The unit tests will help me extend the bracketing functionality to the spatial RSSACRDirect SSA.

### CODE REVIEW

File: src/aggregators/bracketing.jl [hunk 1]
```diff
@@ -35,23 +38,23 @@ BracketData{T1, T2}() where {T1, T2} = BracketData(T1(0.1), T2(25), T2(4))
...
+"Get propensity brackets of constant rate jump."
 @inline function get_cjump_brackets(ulow, uhigh, rate, params, t)
     rlow = rate(ulow, params, t)
     rhigh = rate(uhigh, params, t)
-    if rlow <= rhigh
-        return rlow, rhigh
-    else
-        return rhigh, rlow
-    end
+    return min(rlow, rhigh), max(rlow, rhigh)
```
Comment (isaacsas) [inline src/aggregators/bracketing.jl:53]: Doesn't this do extra work?

File: src/aggregators/bracketing.jl [hunk 2]
```diff
@@ -14,15 +14,18 @@ end
 BracketData{T1, T2}() where {T1, T2} = BracketData(T1(0.1), T2(25), T2(4))
 
 # support either vectors of data for each field, or scalars
+"Get fluctuation rate of species i."
```
Comment (isaacsas) [inline src/aggregators/bracketing.jl:17]: Unless you are writing these to be comments that are visible to users via the REPL, in which case they should document using the standard documentation format that also describes what the arguments are and the function signature, I'd suggest using a hash.

File: test/runtests.jl [hunk 3]
```diff
@@ -2,6 +2,7 @@
 using JumpProcesses, DiffEqBase, SafeTestsets
 
 @time begin
+    @time @safetestset "Bracketing Tests" begin include("bracketing.jl") end
```
Comment (isaacsas) [inline test/runtests.jl:5]: I'd put these lower down the list. Maybe by the composition-rejection table tests.

File: src/aggregators/bracketing.jl (hunk 2 above)
Comment (isaacsas) [reply src/aggregators/bracketing.jl:17]: Any strings you use become visible to users who type "? func_name", whereas I don't think hash-based comments do (so we can think of them as internal only documentation).

File: src/aggregators/bracketing.jl (hunk 1 above)
Comment (isaacsas) [inline src/aggregators/bracketing.jl:53]:
```suggestion
    return (rlow <= rhigh) ? rlow,rhigh : rhigh,rlow
```
if you want it as a one-liner.

File: src/aggregators/bracketing.jl (hunk 2 above)
Comment (Vilin97) [reply src/aggregators/bracketing.jl:17]: Switched to use `#`.

File: test/runtests.jl (hunk 3 above)
Comment (Vilin97) [reply test/runtests.jl:5]: Done. My logic was that unit tests should go first becaues they are faster than integration tests.

Comment (isaacsas) [reply test/runtests.jl]: That makes sense, but we don't currently separate them (perhaps we should), so I'd prefer to keep the tests ordered by how "fundamental" the component is in some sense. If we want to consider restructuring them in the future though that would be a good consideration!

**Comment by isaacsas** [comment]
LGTM. Feel free to merge when tests finished.

**Pull request merged**